### ISSUE TITLE
Weird issue with multi socket CPUs

### ISSUE THREAD

**Issue by OmgImAlexis** [sebhildebrandt/systeminformation#284]
**Describe the bug**
When running `si.cpu()` on a multi-socket server the results are incorrect.

**To Reproduce**
1. Get multi-socket server
2. Run command
3. Profit?

**Current Output**
```js
{ manufacturer: 'Intel®',
  brand: 'Xeon® X5650',
  vendor: 'GenuineIntel',
  family: '6',
  model: '44',
  stepping: '2',
  revision: '',
  voltage: '',
  speed: '2.67',
  speedmin: '',
  speedmax: '',
  cores: 24,
  physicalCores: 3,
  processors: 2,
  socket: 'LGA1366',
  cache: { l1d: 393216, l1i: 393216, l2: 3, l3: 24 } }
```

**Expected behavior**
I think this is the correct result. The server is using 2x 6 core CPUs with 2 threads per core.

```js
{ manufacturer: 'Intel®',
  brand: 'Xeon® X5650',
  vendor: 'GenuineIntel',
  family: '6',
  model: '44',
  stepping: '2',
  revision: '',
  voltage: '',
  speed: '2.67',
  speedmin: '',
  speedmax: '',
  cores: 24,
  physicalCores: 12,
  processors: 2,
  socket: 'LGA1366',
  cache: { l1d: 393216, l1i: 393216, l2: 3, l3: 24 } }
```

**Environment (please complete the following information):**
 - systeminformation package version: `latest`
 - OS: `macOS` and `Unraid`
 - Hardware: `MacBook Pro 2010` and `Dell R710`

**Additional context**
For reference the same version of `si` works perfect on my single-socket Macbook.

```js
{
  manufacturer: 'Intel®',
  brand: 'Core™ i5 M 520',
  vendor: 'GenuineIntel',
  family: '6',
  model: '37',
  stepping: '5',
  revision: '',
  voltage: '',
  speed: '2.40',
  speedmin: '2.40',
  speedmax: '2.40',
  cores: 4,
  physicalCores: 2,
  processors: 1,
  socket: '',
  cache: { l1d: 32768, l1i: 32768, l2: 262144, l3: 3145728 }
```

**Comment by sebhildebrandt** [comment]
@OmgImAlexis can you provide the output of the following:

```
cat /proc/cpuinfo | grep -E "physical id|core id"
```

as well as

```
lscpu
```

Thank you in advance!

**Comment by OmgImAlexis** [comment]
Let me know if you need anything else.

```
physical id     : 1
core id         : 0
physical id     : 0
core id         : 0
physical id     : 1
core id         : 1
physical id     : 0
core id         : 1
physical id     : 1
core id         : 2
physical id     : 0
core id         : 2
physical id     : 1
core id         : 8
physical id     : 0
core id         : 8
physical id     : 1
core id         : 9
physical id     : 0
core id         : 9
physical id     : 1
core id         : 10
physical id     : 0
core id         : 10
physical id     : 1
core id         : 0
physical id     : 0
core id         : 0
physical id     : 1
core id         : 1
physical id     : 0
core id         : 1
physical id     : 1
core id         : 2
physical id     : 0
core id         : 2
physical id     : 1
core id         : 8
physical id     : 0
core id         : 8
physical id     : 1
core id         : 9
physical id     : 0
core id         : 9
physical id     : 1
core id         : 10
physical id     : 0
core id         : 10
```

```
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   40 bits physical, 48 bits virtual
CPU(s):                          24
On-line CPU(s) list:             0-23
Thread(s) per core:              2
Core(s) per socket:              6
Socket(s):                       2
NUMA node(s):                    2
Vendor ID:                       GenuineIntel
CPU family:                      6
Model:                           44
Model name:                      Intel(R) Xeon(R) CPU           X5650  @ 2.67GHz
Stepping:                        2
CPU MHz:                         1595.525
BogoMIPS:                        3191.91
Virtualization:                  VT-x
L1d cache:                       384 KiB
L1i cache:                       384 KiB
L2 cache:                        3 MiB
L3 cache:                        24 MiB
NUMA node0 CPU(s):               0,2,4,6,8,10,12,14,16,18,20,22
NUMA node1 CPU(s):               1,3,5,7,9,11,13,15,17,19,21,23
Vulnerability L1tf:              Mitigation; PTE Inversion; VMX conditional cache flushes, SMT vulnerable
Vulnerability Mds:               Vulnerable: Clear CPU buffers attempted, no microcode; SMT vulnerable
Vulnerability Meltdown:          Mitigation; PTI
Vulnerability Spec store bypass: Mitigation; Speculative Store Bypass disabled via prctl and seccomp
Vulnerability Spectre v1:        Mitigation; usercopy/swapgs barriers and __user pointer sanitization
Vulnerability Spectre v2:        Mitigation; Full generic retpoline, IBPB conditional, IBRS_FW, STIBP conditional, RSB filling
Flags:                           fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pe
                                 bs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 cx16 xtpr pdcm pcid dca sse4_1 sse4_2 popcnt aes lahf_lm pt
                                 i ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid dtherm ida arat flush_l1d
```

**Comment by OmgImAlexis** [comment]
These are identical CPUs as well. You may want to display the output differently for multi-socket systems as they may have different CPUs in each socket.

**Comment by sebhildebrandt** [comment]
@OmgImAlexis ok, I know where the problem is ... will provide a solution mid of next week as I need to finish a project till Tuesday

**Comment by OmgImAlexis** [comment]
No worries at all. Thanks for the quick updates as usual.

**Comment by sebhildebrandt** [comment]
@OmgImAlexis I changed the code now (not yet released on npm). Can you test it on your side? you only need to grab `lib/cpu.js` from github and replace your existing one. If this returns good results, I will publish this on npm. Thank you in advance!!

**Comment by OmgImAlexis** [comment]
New output looks better.

```
{ manufacturer: 'Intel®',
  brand: 'Xeon® X5650',
  vendor: 'GenuineIntel',
  family: '6',
  model: '44',
  stepping: '2',
  revision: '',
  voltage: '',
  speed: '2.67',
  speedmin: '',
  speedmax: '',
  cores: 24,
  physicalCores: 12,
  processors: 2,
  socket: 'LGA1366',
  cache: { l1d: 393216, l1i: 393216, l2: 3, l3: 24 } }
```

**Comment by sebhildebrandt** [comment]
@OmgImAlexis - thank you for testing! New version published on npm v4.14.12
